templates/cicd/buildspec.yml

```diff
@@ -0,0 +1,74 @@
+# Buildspec run in the build stage of your pipeline.
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      docker: 18
+      golang: 1.13
+      nodejs: 10
+      ruby: 2.6
+    commands:
+      # Download the CLI.
+      - export GOMOD=$CODEBUILD_SRC_DIR/go.mod:GOMOD
+      - export GOPATH="/go"
+      - echo "cd into $CODEBUILD_SRC_DIR"
+      - cd $CODEBUILD_SRC_DIR
+      - git clone https://github.com/aws/amazon-ecs-cli-v2
+  pre_build:
+    commands:
+      # Build the CLI.
+      - cd ./amazon-ecs-cli-v2
+      - make tools
+      - make release
+      - echo "Built artifacts:"
+      - ls -lah ./bin/local
+      - cp ./bin/local/archer-amd64 ../archer
+      - cd ../
+  build:
+    commands:
+      - echo "Run your tests"
+      # - make test
+  post_build:
+    commands:
+      - ls -l
+      - export COLOR="false"
+      # Find all the local applications in the workspace
+      - manifests=$(find ./ecs-project -name '*-app.yml')
+      # Remove forward slashes and the trailing -app.yml to get the names
+      - apps=$(find ./ecs-project -name '*-app.yml' | sed -e 's!.*/!!' -e 's/-app.yml//')
+      - envs=$(./archer env ls)
+      # Generate the cloudformation templates.
+      # The tag is the build ID but we replaced the colon ':' with a dash '-'.
+      - tag=$(sed 's/:/-/g' <<<"$CODEBUILD_BUILD_ID")
+      - >
+        for env in $envs; do
+          for app in $apps; do
+          ./archer app package -n $app -e $env --output-dir './infrastructure' --tag $tag;
```

Comment:
As of now ]the pipeline CFN deployment actions try to locate (https://github.com/aws/amazon-ecs-cli-v2/blob/master/templates/cicd/pipeline_cfn.yml#L284) the artifacts directly under `CODEBUILD_SRC_DIR` instead of `CODEBUILD_SRC_DIR/infrastructure`. If we want to change the outputdir then could you also update the expected template location?

Reply:
Nice thanks for pointing this out! Done